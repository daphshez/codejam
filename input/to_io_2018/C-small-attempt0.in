100
3
BCB CAB CBC
2
CC CA AA
2
AA AB CC
2
AA AB BA
2
AA BA CB
100
BAABBBAAAAAABAABBABBBBBBBABAAAABAAAABAABABBABBBAAABBBBABBBBAABBAAAABABBBAABBAABAABBBAAABBABBBBABBABB BAABABAABAABBABBABABBABBBBAABBBBBAAABBABBBAABBBBBBAABBBAAAAABABBABBBABABABBBAAABABABABABBABBAAAABBAA AABBBBAABBBBABBABBBABABAABAAAAABBBAAAAAABABBBAAABBBBAABAABAAAABBABBAAAABAABBABAAABBABABBBBABABAABAAB
100
BCBCBBBBCBCBCCCBBCCBBCBCCBCCCBCCCBBBBBBCCCCBCBBCCCCBCCCBCBBCBCBBCCBBCBCBCCBBCBCCBBCCCBCCBCBBCBBBCBBC BBCBCBBBBBCBBBCBBCCBBCCCCCBCBBBBBCBCCCBBCCBCBBCBCBCCCBCCCCBCCBBBCBCCCCCBCCBCCCCBBBBCBCBBBBCCBCCBBCCC BBBCCBBCCBCCBCCCBCCBBBBBBCBBBCCCCCBCCCBCCCBCCCBCCCBCBBCBCBBBCCCCCCBBBBCCCBCBCCBBCBCCBCCCCCCBBBCCBCBB
2
AB CB CC
2
AA BA CC
10
BBBCCBABBA ABACABBCBB ABABCCACBC
100
ACBBCBCACABCBBABACABABCCABACBCAABCCBBBBBACCCCCAABBCACBCCACCCBCAAAACBCAAAACCAAABBBBBBCAAAAAACAAABBCAC AACCBCCBACABAACABACCACBBAACACCCABBAABACBCCAAACCCABCCAAABBBBBBBCCACCBCBACCBACBCBCAABBCAABCCCABBACBAAB ABBCABCCBCCCACBCCCBCCBCABACCBBACACBBBAAAABABACBABACBCBCAACACBBBBBAAABBBCACAABBBBCAABACCACCCABACAAAAB
100
ABCABCABCABCABCABCABCABCABCABCABCABCABCABCABCABCABCABCABCABCABCABCABCABCABCABCABCABCABCABCABCABCABCA ABCABCABCABCABCABCABCABCABCABCABCABCABCABCABCABCABCABCABCABCABCABCABCABCABCABCABCABCABCABCABCABCABCB ABCABCABCABCABCABCABCABCABCABCABCABCABCABCABCABCABCABCABCABCABCABCABCABCABCABCABCABCABCABCABCABCABCC
2
AA AB BB
2
AB BB CC
2
AA BB CA
41
BAAACCCCAAAACBCBCBBAACACABAAACBBBAAAACCAA CAABACBCBCCAAAABABCAABBBBABBCCBAACCCCBBBA BBBCCAACAAABABABCBBCBBBACAAABBCAABCBCCACB
2
AB CA CB
83
ABBBCACABAACCACCBABABCCAACBBBCBCBAABCACABACBABACCCABACAABBCABBCCCAABBABABBBBACACCBB ABCABAABBACBCACBBCCCBBAABABAAACBACBBCAABAAAABCBAABABCBCACBBABCBCBBACCCBCCCABCBAAAAC CCBACACAABCCBBBBAACAACACCCCCCACBACCCABBABBBACCBAAAAABBABAABABBBCBBCBCAABBCBBBCCCBAA
2
AB BC CC
15
CABACABBABBCAAA AACABABBBCBCABC ABCCCCAACACACCC
2
AA AB CA
28
AACACABBCCCAACAAABBBACACBCBA AACACBBBAAACCCACBBACAABBCCBB BCBCBBAABCBBCBACCABACCAACCCC
2
AA BB CC
100
CCBBBCCCBCBCCCBCBBBCCBBBBBCCBBCCCCCCBBCCBCBCBBBCBCBBCBBBBCCBCBCBCBBBBCCBBCBBCBCBCCBCCBCCCBCBBBBCBBCC CBCCCCBCBCCCCCBBCCBBCCBCBBCBCBCBCCBBBBCBCBBBCBCBBCBCBBCBCBCBBBBCCBBBBCBBBBBCCCBCBBBCCCCBCBBBCBCBCCBC CCCCCBBBCBCBCBCBCBBCCBCBBBBCBCCCBBBCBBBBCCBBCCBCCBBBBCBBCCBCBCBBCBCBCCBBCBBBCBCBBCCBBCBBBBCBCBBCBBCC
100
AACBCBBAACCBBABBBBABCCAABCCACABBBBCBCCCABBBBBCCBBBABAABCCBABCCBAAACACBBCAAACAABCBBAACACABACACABBCBCA AACBCBBAACCBBABBBBABCCAABCCACABBBBCBCCCABBBBBCCBBBABAABCCBABCCBAAACACBBCAAACAABCBBAACACABACACABBCBCC CACBCBBAACCBBABBBBABCCAABCCACABBBBCBCCCABBBBBCCBBBABAABCCBABCCBAAACACBBCAAACAABCBBAACACABACACABBCBCA
100
BBCCCCBCCBCBCBCCCBCCCBCCCCBBBCBBCCBBCBCBCCBBCBCBBBCCCBBBBBBBCBCBBCCCCBCBCCBBCCCBBBCCCBCCBCCBCBBCCCBC BCCCCBBBCCBCCBBBBCCBBCBBCCCCBBBBCCBBCBCCCCBBCCCCBBBCCBCCCBBBBBBBCBBCBCBCCBBBCCBBCBCCBBBBBBBCBBCCCCBC CBCBCCBBBBBCBCBCCBCBBCCBCBBBCBBBCCBBBCBBCCCBBBCBBBBCCCCBCCCBBCCCBCCBBCBBBCBCCCCBCCCBBBBCBCCBBBCBBBBB
42
CCCBBCABAACCCABCABABCCABABABACBCABBBBBCCCB CCBABCCABCCCAABBACBBCCCAABAABABACABBABBCAC BACCCACCBCBBBCABABCACBABABCCAACAABABCAABAB
100
CBABBBACBABBCBCAACBACAABCAAACACAAABBCABBABAACACCCABBABAACABACAACBBBBCCAAABBABACBBABBABCACAABBABAACCA BABCBCAAACCBBAACCCCCCBBABCBCBAAABBCCBAABAACCBCCABBABCBBCBACCBCBAAAACCCACCCCACBCBAACBCCBBAABBACCBAABC CBCCCCACBABCBABBACCACBCABBCCCBCCAACAAAACBBBCCCAACBABCACBACBAABABCACAACCCBBCABBBACABCABBABCBBCCABCABB
2
AA AB CB
2
AB BB BC
100
CCACACABAACBBCBACCCBCCCBBBBCABBAACCAAACABCCCCBACCAACACBBBCBBCCCCABAACCBBCBBBACBAABAABBCABBCBACBBCCBB CAACACABAACBBCBACCCBCCCBBBBCABBAACCAAACABCCCCBACCAACACBBBCBBCCCCABAACCBBCBBBACBAABAABBCABBCBACBBCCBB ACACACABAACBBCBACCCBCCCBBBBCABBAACCAAACABCCCCBACCAACACBBBCBBCCCCABAACCBBCBBBACBAABAABBCABBCBACBBCCBB
2
AA BC CC
100
AAABAAABBABBABBBBABBBBBBBAAAAABAABBABAAAABBBAAABABAAABBBABBABAABBABAABAAABBAABAAAAABABAAAAABAAAAAAAA BBBAAAAAAABABBBBBABBABBABBAABBABABBAABBAAAABBAAAABAAABABABBAAABABAABBBBBBAABAABABBABAABBBABBAABAAABB BBBBBBABABBBBBBBBAABABABBABBAABBBAAABABBAABBBAABABAAAABAAABABAAAAABABABBBAAAAABABBBAAAAABBBAABAABBAA
100
CCACABBBCBCACCCBBCBAABBBCCBCBBCBBCBAAABBBBBBABAAACBBACCACBCAABCCBBAACBBACACBCABCCCBACBBBBABCCBBAACAA CCACABBBCBCACCCBBCBAABBBCCBCBBCBBCBAAABBBBBBABAAACBBACCACBCAABCCBBAACBBACACBCABCCCBACBBBBABCCBBAACAC CCACABBBCBCACCCBBCBAABBBCCBCBBCBBCBAAABBBBBBABAAACBBACCACBCAABCCBBAACBBACACBCABCCCBACBBBBABCCBBAACCA
5
ABABA CCBAC BCCBC
100
BBAABAABBBAAAABBBBBAABBABBAABABABBABABABABAABBBAAAAABAAABBBBBABAAABABABBBBBAAAABBBBABABABABAABAABAAA AABBBABAABBBAAABABABBBBBAAABAABAAABAAAABBBAABABABBBABBABBABBAABBBABBABABABBBBBBBABBBABBBBAAAABBBBABB BBABAAAABABBBABBAAABAABABAAAABBBABABBBBABAAAABBABBABBBBAABAAABABAAAABBAABAABBBAAAABBBBBBABABAAABAABA
2
AB BC CB
2
AB AC BC
100
CACAAACBACBCABAABAACAACBCBBABCAABABBABABABABABBBCCBACACBBCCBBABCAABACBACBABBAABAACAACAABAACBBCCBABAA BAACBABBACCACACCBCBBCBAABAAACAAAABABBCBCAAACBCCACBCCABACBAABABBBCCBCCCBCACCAABACABCBCABAABCCCBACBCBB BABAAAABABBACBBBCCAABCCABBAAABBAACBABCCBABBBCBACAAAAAAACBBCABCBBCCCABCBBAAABBBACBBAACBABACBCCCBCBACA
10
ACBBABCACC CCAAACCCBB AAABCBCABA
77
BBBCCCCCCBABCAACABBAAACBACCACAACABCACCACAABAACCCBBABACABCCBAACCBBBBBBBBCAACBA ABCCCBCCBBABCAACABBBAABCCAACACBBAABACBBBABBACBACCCBAACBCABACBBAACCBCCCBCBACAB BABACCACBABCAABCBCBBABBAAABABABCAACACABABCBCCABACACCBAACCACAABCBCCACABBBAABAA
2
AA BA BB
100
CBCCBCBCCCBCBBCCBCBBBCCCCCBBCCBCBCBBBBCCCCBBBCCBBCCBCBCBBCBCCBBCCBCCCBCBBCBBCCBCCCBBBCBBCCCBCBCCBBBC BCBBCBBBBCCCBCCBCCBBBBBBBCCBCCBCCBBBBBBBCCCBCBBBBCBBCBBCCCCBCCCBCCBBBBBCCBCBBBBBCBCBBCCCCBCBBCBCBBCB BBBCBCBBCBCBCBBCCCCBBBBBCBCBBCBBCBBCBBCCBBBCBBCBBBBBBCBBCCBBBBBCCBBCBBBCBBBCCCCCCCCBBCBBBBBCBBBCCCBC
2
AA BB CB
100
ABACCACABBACABACACCABBCBABCACAACCBACBAACABCACCABCBCBCBACACCAABABCBBBCBBCABAABBCCCBAACAAAAAACCCBCACAA ACABBACAACCAAAAACBACCCCCBABBCCABBAAACABCCBACBBABAABAABCBCABCCBABCBCBBABACCACBAABCCCCBCCAABAAABBCCBBC CCCABBCCCCBBCBCBACCBACBBBAACCCBCCBCBCBCCCBBCABBBAAABCBCCCBBCACAABCABCCBCACBBBCACAACACCCAAACCACCCACBA
66
BBBACBBCBBBBBABBACCAABACCBCABABBCACCBACAABBBBAACAACBCABCCBCBBBABBA AABAABCCABCBBABBCACAACCAABCCAACCABCBACAABCBCAAACCABCCBABCCAAAACBBB CCCCAAACABBBBACCACACBABCAACABCCBBBBCBBCBCABBBABCACCCACBBCCBACACACC
100
BBCAACACACBCABBCCBCABCAAABABACBCBAABCAAAACAAAABABBAAAABAACCABAACBACBBACCAAAABCBAACABACBBCABBBCAAAACA BBCAACACACBCABBCCBCABCAAABABACBCBAABCAAAACAAAABABBAAAABAACCABAACBACBBACCAAAABCBAACABACBBCABBBCAAAACC BBCAACACACBCABBCCBCABCAAABABACBCBAABCAAAACAAAABABBAAAABAACCABAACBACBBACCAAAABCBAACABACBBCABBBCAAAAAA
100
BCBBCCCCBCCCBBCBBBCBBCCBCBBBBBCBCCBBCCBCCCBCBBCBBCBBBBBCBBCBCBBBBBCCCCBBBCBBCCCCBCBCCBBCCBCBBCBBCCCC BBCCCCCBCBCCBCBCBCCCCBCCBCBCCCBCCBBBCBBBBBBBCBBCCCBBCCBCCBCBCCCCCBCCBBCBBCCCBCCCBCBCCBCBBBBCCCBCCBBC BCBBCCCCBBBBBBCCBBCBBCBBBBCBBCCBBCBCBBBBBCBBCBCBBBBCCBCBCBBCCCBCCBCCBCBCBBCBBCBCCBBCCCCBCCCBBCCCBBCB
80
BCCBBABCAABABBBCBACBBAAABBACAACACBBBAAAAAACACACAACAACABBCCBAABAAACCAAABBAABABCBA BBBCACACACABBABBBBCCCAAACAAAACABBCBBACCABCCCBBBCAABACCABABBBBABACCCCBBABBCBCCBCC BBABCACACCCAAACACBBCABBCCCCBACBABBCBBCBCCBACCCCAAACBABBAABCCAAACCCABCAACBBBBACBC
100
BBAAAABBBABABBABAABBBBBBAABBAAABBABBABBBBAAABAAABBBABBBBBABBBBBABAAABBBABBBBABBBAABBBBBBBAAABBBAABAB AAABBBABABAAAAABBBAABBBAAAAABBABBAAABBABAABAABABAAABBBBABAAABABAABABAAABBABBBBBAABBBABBBBABAAAABBAAA BBBABABAAAAABABBBBAABBABBAABABBBAABBABBABAABAAABBBBAABBABABBBABBAAAABBBBBBABBAAABBAAAAABABBBBAABAABA
2
AA AB BC
2
AB BA CB
100
CCCBBBBCCCABBACBCACBCBAAACCACAABAACACBCCBBCCBBBAABCCAABACABABABCABCAACABAACABBBCABCACACABABABACCBCAA CCCBBBBCCCABBACBCACBCBAAACCACAABAACACBCCBBCCBBBAABCCAABACABABABCABCAACABAACABBBCABCACACABABABACCBCAC ACCBBBBCCCABBACBCACBCBAAACCACAABAACACBCCBBCCBBBAABCCAABACABABABCABCAACABAACABBBCABCACACABABABACCBCAA
100
CCCACACBCCCCBBACBAAABBAAACBCCCCCBCABCAACCCACBABAACAAAABBBCBBABACABBCBBACBABCACACABAACBCBBBCBCABACAAC CCCACACBCCCCBBACBAAABBAAACBCCCCCBCABCAACCCACBABAACAAAABBBCBBABACABBCBBACBABCACACABAACBCBBBCBCABACAAA ACCACACBCCCCBBACBAAABBAAACBCCCCCBCABCAACCCACBABAACAAAABBBCBBABACABBCBBACBABCACACABAACBCBBBCBCABACAAC
2
AA BA BC
2
AB CA CC
17
BCBCBCBBCBBBCBBBB CBBCCCBBCCBBBCCCC BCBCCCCBCCBBCBCCB
100
BCCACCACCBBCAAAACCAABCBACCBABBCACCABBBCBAAAABCBBBCCABBCCAAAAACACBCABAAAABBBAAABBABCCCCBBCBBCACBABACC BCCACCACCBBCAAAACCAABCBACCBABBCACCABBBCBAAAABCBBBCCABBCCAAAAACACBCABAAAABBBAAABBABCCCCBBCBBCACBABACA BCCACCACCBBCAAAACCAABCBACCBABBCACCABBBCBAAAABCBBBCCABBCCAAAAACACBCABAAAABBBAAABBABCCCCBBCBBCACBABAAC
100
ABBCCABBBCBBCBACBABABBBBABCCAABCBABABBCCAABBCBBACAABCCACBACBBBCCBBACCCCBAACCCBACBBBACBAACAAABCAAAACC ABBCCABBBCBBCBACBABABBBBABCCAABCBABABBCCAABBCBBACAABCCACBACBBBCCBBACCCCBAACCCBACBBBACBAACAAABCAAAACA CBBCCABBBCBBCBACBABABBBBABCCAABCBABABBCCAABBCBBACAABCCACBACBBBCCBBACCCCBAACCCBACBBBACBAACAAABCAAAACC
2
AB AC CA
17
AABBAACBABBABCCBB CBCCBABBBBCBCAACC CACBCCBBBCBACCAAC
2
AB BA BC
2
AB BB CB
2
AB BA CA
100
CBBCBCBBBCBBCBBBCCCCCBCBBBCCCBBBBBCBCCCCCCBBCBBBCBCBBCBBCCCBCCCCBBBCBCCCBCCCCBCBBBBBCBBCCCCBBBCBCCBC BCBBBCCCCBCCCCCBCCCBBCBCCCCBCCCCBCBBCCCBBBBCCCCBCCBCBBBCBBBCCCBBCBBBBBBBBBCBBBBCCCBCCCCCCCBBBCBCCCCC CBCCBBCBCBCBCBBBCCBBBBBBCCBCBCBCCBBBBBBCBBBCBBCCCCCCBCCCBCBBCCCBCBBCBCCBCCCCCBBBBCCBBBCCBBCCCCCBCBCC
4
AAAB CBAB AAAC
100
BBCCBCCCBCCCBBCBCCCBCBBCCBCBBBCBBBCBBBCBCCCBCCBBBCBCCBBBCBCCBBCBBBBBBBBBCBBBCBCCBCCCBCBCCCBCCCCCCBBC BBCCBCCCBBCBBBCBCCCCBCBCBBCCBCBBCBCBCBCCCBCCBCCBBBCCCCBCCCBCBBCBBBBBCCCBBCCBBBBBBCCCCBBBCBBBBCBCBBCC CBCCCBBCBBBBBCCCCCCCCCCCCCBCCCBBBCBCCBCCBCCBBBCBCCCBBBCCCBBBCBBCCBCBBBCCCBCCBCCCBBCBBCBCCBCBCBBBBCCC
100
CACCBBCBACBACAABACABBBACCBBCBBBBCAABCAAABCAAACBBBBAABACACBABCCCCACBAABBBBABCCCABCBCAABBCACAACAAAABAB CCCCBBCBACBACAABACABBBACCBBCBBBBCAABCAAABCAAACBBBBAABACACBABCCCCACBAABBBBABCCCABCBCAABBCACAACAAAABAB AACCBBCBACBACAABACABBBACCBBCBBBBCAABCAAABCAAACBBBBAABACACBABCCCCACBAABBBBABCCCABCBCAABBCACAACAAAABAB
67
CCCBABBACBCBBCBBCBAABCACBACCBACBACAACCABAACCABBACCABCCAACACBBCABAAB CACBACBAACCACCBABABAAAAABBAACBCCBCACCBAAAAAAAAAABBACCBCBAACCCBBABCA BABBBCABCACCBCABBABCACBABBCCACABACBABCBCACBCBCBCCACABBBCABABBBACACC
100
AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAABA AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAACB AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAABC
86
BAABCABCCCBCBCBCCCBABABCABCCACBACBBABAABCBBBBAABAACCBAABBCCCAACACBBBCCCABBBBCACCBBCCCC BBCAACBBBBCCCACAACBCABCCBBBAAAAABBBCBBACCBCABCABBCBBBBBAACBBAACCBACABCACBACCCCBCCABBCA CCCABBBAAABCBAABCBCBCCBACBABACBCCBAACCBACAABABABACCAACACABAABBABAACAABBBCCCABABCACCABA
100
AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAC AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAB AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
100
AACCABAACBCBCBACCACCACBAABBBBCACBACCCABACABCBBBBACBACCBCBABCACBCABCACABBAAACBAABABCCBCAACCABCAAABCBB ACCCABAACBCBCBACCACCACBAABBBBCACBACCCABACABCBBBBACBACCBCBABCACBCABCACABBAAACBAABABCCBCAACCABCAAABCBB CACCABAACBCBCBACCACCACBAABBBBCACBACCCABACABCBBBBACBACCBCBABCACBCABCACABBAAACBAABABCCBCAACCABCAAABCBB
100
ABAABCCBCBBCBBBBCBABCACBBBCBBBCCBCAABCCBBABBAACCBABCABABAAACACBBCCBBAAABCBACCACAABACCBBACCACCCACBCBB ACACBACBBBCACAAACBBBBBCBACABCCBACBCBCCCCBCCCCACBBABBCAACABBABCBBBBACBCCACCCCCCBAAACCACCBBBCABABABBCA BCBABAAAAACAABACBCACAAACABACAACCBAACBBACCCCCAACCCACBACBBACBBBBACABBCABCCBCCAACCAACBBBACACACAAACACCCA
2
AB AC CC
34
AACABCCAACAABBBACBBBCCCBACBCBBACAB ABBCCCCAABCCBBBCABCCACCBCCABCAACCB CCBCACACCBBABBCACCACBACCBBACBAABAC
2
AB BC CA
85
BBBBBCCCBCBCBBCCCBCBCCCBBCBBBBCBCCBCBBCBCCCBBBCCBBCBCBBCBBCBBBBCBCCBCBCBCBBCCCBCCBCBB CCBCBBBCCCCBBCCBBCBCBCBCBCCBCCBBCCBCCBBBCCBBCCBCCBBCBBBBBCBCCBCBCCCCCBCCCBCBCCBBBBCBC BBCCBBBBCBBCBCBBBCCBBBBBBBBBBBCBBBCBBBBBBBCBCCBBCBBCBBCBCCBCCBCCBBCCBBCCBBCBCBBBBBBBC
100
AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA BBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBB CCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCC
2
AB AC BB
100
ACBACCAACAABBCBAACCCAACAAACABACCCBAABAABCACAAABBAABACBBCBBBABAACCACABCBBBACCBBBBCACBCBBBAAACAABABBAC AABACCAACAABBCBAACCCAACAAACABACCCBAABAABCACAAABBAABACBBCBBBABAACCACABCBBBACCBBBBCACBCBBBAAACAABABBAC CCBACCAACAABBCBAACCCAACAAACABACCCBAABAABCACAAABBAABACBBCBBBABAACCACABCBBBACCBBBBCACBCBBBAAACAABABBAC
100
CABCBCBCCBAAACAAACACACACCCABCBCBCCBBBAABBACCAACBCCBAABAAABBAABBCAABCBBACCACABCCACABABBAAACABAABBAAAB CCBCABABBABBABBAABBBBCCBCBCACBCBABBAAABBAAABBBCAAABBBBBAAACACBBCCBCBBBACCBABAABBBBACABBBAABCBACCBCBA BBCBCAABBAAABABBAACAAAACBACBBABAABBACCBBCBBBACCACCBAABBCABABCCBAAAAAAABBCACCAACBCBBABAABCBACACCCAACA
93
CBCCCCCBCCCCCBCBCCCBBCCCCBCBBCBBCCCBBBCBBBCCCCCBBBCBCBCBBBCCBCCCBBBBCCBCCBBBCCCCCBCBBCCBCBCCC BCCCCCBBBCCBCBCBCBCCCCCBBCBBCCCCBBBCBBCBCCBBCCBBCCCBCBCCCCBBBCBCBBBBCBBBCBCBCCBBCCBBCCCBBCCCC CBCCBCBBCCCCBCBCCBBBBBCBBBBCCCBBCCCBBBBCCBBCCBBCBCCCCCCCCCCBCCBBCCBCCCCBCCCBBBCBCCCBBBBCBCCCB
2
AA BA CA
2
AB BA BB
2
AB AC BA
60
BBACCBCBABACCCBABCBABAAACCCBACACBBCCBBABABBABAACCCBBCBBCABBB BCBBBAAACACABBBCBCABBBABABBAAAACCCABAABCBBBACAACCABCCCACBCBC CBCABCBBABABCBCCCCCCBBBABCBAACCABCACBAACBBBABAAAAACCCBABCBBC
2
AA BC CA
2
AB AC CB
100
CCBBBBCBBCBCCBBCCBCBBBBCCBCBCBCBBCCBCBBBCBCCBCBBCCBBCBCBCCBBCCBCBCBCCBCCCCBCBBCBCBBBBCBCBBCBCBCCCCBC BBBCCBBCBBCCBCCBCCBCCCCBCCCCBCBBCBCBBBBBBCCBCBBBCBCBBCBCBCBBBBCBBBBBBBBBBCBCCCCBCBBCCBCCCCBBBCBBCBCB CCBCCCBBBCCCCCCCBBBCCBCCCCBCBCCCBCBCBBBBCCBBBCCBCBBBBCBBCBCCCBBCCBCBBCBCBBCBCCCBCCCCBCCCCBBBBBCBBBBB
2
AA BC CB
22
BCBCCCBBABACCBBBBCCBCC BCAACCACCCBACCABBBACAA ACBBCBBCACCCABBAABBACA
2
AB BB CA
100
AABAAAABABBABAAABBBBAABBABAABBABBBBBAABABAAABABABABBBAABBAAAAAABBBBBBABAABBBBABAABABBABBBBBBBBAABBAA BBBABABAABBAABABBBAAAABBBABABBBBBABAAAAABBABAABABBABBAABAABBBAAAABAAABABABBBBABBBBABBBABBBBAABBABBBB ABABABAABBBBABAAAABBBAABABABABABAAABAABBBAABBABAABBABBBBABABAAABBAABABABBBAABAAABAAABBAABABBBBAABABB
1
C B A
2
AA BB BC
2
AA AB AC
2
AB BA CC
100
CCABCCBCCACCCBBACCABCCCABACBAABACCCCCBABBCBABBBABBCACCBACACABABBBBBACBAABBBCCCAACACCCACACBAAACBCABCA CABBCCBCAABAAACABCBABCCCCBBACCBCCCBCACCBCBBACBBAAACBCBBBCAACCBCAAACBACCAAABCBABCABCACBACBACBBBBCCBBA BABCCAABABBBABBCCBCABACACBACBCACCBBBBBBCACAABCAABBBABCBABBCCBACACBABABACBBACBBACCCCBBBBAACABCACCCAAC
100
ACAACBABCBBCAAACABABAACCACACABBAABCAACAABBACCBBACCCACBCBBABBCAAACCBACBBBCCBAACACCCCCBCACAABAACBCBCAC ACAACBABCBBCAAACABABAACCACACABBAABCAACAABBACCBBACCCACBCBBABBCAAACCBACBBBCCBAACACCCCCBCACAABAACBCBCAA ACAACBABCBBCAAACABABAACCACACABBAABCAACAABBACCBBACCCACBCBBABBCAAACCBACBBBCCBAACACCCCCBCACAABAACBCBCCC
