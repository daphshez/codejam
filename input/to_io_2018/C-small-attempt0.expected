Case #1: NO YES YES
Case #2: NO YES NO
Case #3: YES YES NO
Case #4: NO YES NO
Case #5: YES YES YES
Case #6: NO YES NO
Case #7: NO YES NO
Case #8: NO YES YES
Case #9: YES YES YES
Case #10: NO YES YES
Case #11: YES YES YES
Case #12: YES YES YES
Case #13: NO YES NO
Case #14: YES YES YES
Case #15: YES YES YES
Case #16: YES NO YES
Case #17: NO YES YES
Case #18: YES YES NO
Case #19: YES YES YES
Case #20: NO YES YES
Case #21: YES YES NO
Case #22: NO YES NO
Case #23: YES YES YES
Case #24: YES NO NO
Case #25: NO YES NO
Case #26: NO YES NO
Case #27: NO YES NO
Case #28: YES NO YES
Case #29: YES YES NO
Case #30: NO YES YES
Case #31: NO YES NO
Case #32: YES YES YES
Case #33: NO YES NO
Case #34: NO YES NO
Case #35: YES YES YES
Case #36: YES NO NO
Case #37: YES YES YES
Case #38: YES YES NO
Case #39: NO YES YES
Case #40: YES NO NO
Case #41: NO NO YES
Case #42: NO YES NO
Case #43: NO YES NO
Case #44: YES YES YES
Case #45: YES YES NO
Case #46: YES YES YES
Case #47: YES NO NO
Case #48: NO NO YES
Case #49: NO YES YES
Case #50: YES NO NO
Case #51: NO YES NO
Case #52: YES YES YES
Case #53: YES NO NO
Case #54: NO YES NO
Case #55: NO YES YES
Case #56: NO YES NO
Case #57: NO NO YES
Case #58: NO YES NO
Case #59: YES NO NO
Case #60: YES YES NO
Case #61: NO YES YES
Case #62: NO YES YES
Case #63: YES YES YES
Case #64: YES YES YES
Case #65: YES NO NO
Case #66: YES NO YES
Case #67: YES NO NO
Case #68: YES NO NO
Case #69: YES YES NO
Case #70: YES NO YES
Case #71: YES YES NO
Case #72: YES YES YES
Case #73: NO YES NO
Case #74: YES YES NO
Case #75: YES YES NO
Case #76: YES YES NO
Case #77: YES YES YES
Case #78: NO NO YES
Case #79: YES YES YES
Case #80: YES YES NO
Case #81: YES NO NO
Case #82: YES YES NO
Case #83: NO NO YES
Case #84: YES YES YES
Case #85: NO YES NO
Case #86: YES YES NO
Case #87: NO YES NO
Case #88: YES YES YES
Case #89: YES YES NO
Case #90: YES NO NO
Case #91: YES YES YES
Case #92: NO YES NO
Case #93: YES YES YES
Case #94: NO NO YES
Case #95: YES YES YES
Case #96: NO YES YES
Case #97: YES YES YES
Case #98: YES YES YES
Case #99: YES YES NO
Case #100: YES NO NO
